src/types/resolveExpression.ts

```diff
@@ -562,13 +562,6 @@ function resolveCall(
 
     // Handle ref
     if (src.kind === "ref") {
-        if (src.optional) {
```

Comment:
So, this change will allow optionals in not only `extends mutates` functions, but also in `extends` functions, right? Maybe we should add a test case of an `extends` function as well.

Reply:
resolved